dev-docs/activity-controls.md

```diff
@@ -51,16 +90,27 @@ pbjs.setConfig({
 | `rules[].allow`    | Boolean | Whether the activity should be allowed when this rule applies. Defaults to true. |  
 | `rules[].priority` | Number | Priority of this rule compared to other rules; a lower number means higher priority. See [note on rule priority](#priority) below. Defaults to 1. |  
 
+`Rules` is an array of objects that a publisher can contruct to provide fine-grained control over a given activity. For instance, you could set up a series of rules that says:
+
+- Amongst the bid adapters, BidderA is always allowed to receive user first party data
+- Always let analytics adapters receive user first party data
+- otherwise, let the active privacy modules decide
+- if they refuse to decide, then the overall default is to allow the transmitting of user first party data
+
+There's more about [rules](#parameters) below.
+
 <a id="activities" />
 
 ### Activities
 
+Here's the list of 'potentially restricted activities' that Prebid.js core can enforce for Publishers.
+
 {: .table .table-bordered .table-striped }
 | Name           | Description | Effect when denied | Additional parameters |
 |----------------|-------------|---------------------------|--------------------------------|
 | `accessDevice` | A component wants to use device storage  | Storage is disabled | [`storageType`](#params-accessDevice) |
 | `enrichEids` | A user ID or RTD submodule wants to add user IDs to outgoing requests | User IDs are discarded | None |
-| `enrichUfpd` | An RTD submodule wants to add user first party data to outgoing requests (`user.data` in ORTB) | User FPD is discarded | None |
+| `enrichUfpd` | A RTD submodule wants to add user first party data to outgoing requests (`user.data` in ORTB) | User FPD is discarded | None |
```

Comment:
I'm sure you won't mind if I  teach you your native language in the most annoying way possible :)

https://www.google.com/search?q=an+or+a+in+front+of+acronym

> When the indefinite article is used before an acronym, the choice of form (a or an) depends on pronunciation, not on spelling; in other words, use a if the acronym begins with a consonant sound, and an if it begins with a vowel sound: a NATO decision (a before the consonant sound n)

Reply:
Fixed. :-)